DependencyInjection/CompilerPass/LegacyPass.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Don't need a `CompilerPass` for this. You can do it directly on bundle extension.

Sample here: https://github.com/sonata-project/SonataAdminBundle/pull/2963/files#diff-d81fe8cfe0444cbcab9e87a76c5e9970R134